### PR TITLE
Terraform Kickstarter @singergs

### PR DESCRIPTION
**Pull request by singergs**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

The following supports the creation of a Prowler resources using terraform.  The modules support Prowler running inside an AWS Codebuild project with the results being sent to AWS SecurityHub.

The terraform will activate SecurityHub along with the integration to Prowler.  See readme.md for all resources created when deploying.

### CODE REVIEW

**Comment by toniblyx** [comment]
Thank you @singergs, I'll review this next week.

File: terraform-kickstarter/prowler_build_spec.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      - curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+      - unzip awscliv2.zip
+      - ./aws/install
+      - git clone https://github.com/toniblyx/prowler
+      
+  build:
+    commands:
+      - echo "Running Prowler as ./prowler $PROWLER_OPTIONS"
+      - cd prowler
+      - git checkout 2.5
```
Comment (w0rmr1d3r) [inline terraform-kickstarter/prowler_build_spec.yml]: I'd move these lines, together with the `git clone` from above to the pre-build phase instead.
At the same time, why are we forcing 2.5, what will happen when a new version appears?

File: terraform-kickstarter/main.tf [hunk 2]
```diff
@@ -0,0 +1,535 @@
...
+  default     =  "-q -M json-asff -S -f us-east-1"
+}
+variable "prowler_schedule"{
+  description = "Run Prowler based on cron schedule"
+  default="cron(0 0 ? * * *)"
+  type=string
+
+}
+
+variable "codebuild_timeout" {
```
Comment (w0rmr1d3r) [inline terraform-kickstarter/main.tf]: should we separate the variables and data to other files?

File: terraform-kickstarter/prowler_build_spec.yml (hunk 1 above)
Comment (singergs) [reply terraform-kickstarter/prowler_build_spec.yml]: Thanks for the comment and made the changes as suggested.

File: terraform-kickstarter/main.tf (hunk 2 above)
Comment (singergs) [reply terraform-kickstarter/main.tf]: I have this as a TODO.  Thanks for the comment!

Comment (singergs) [reply terraform-kickstarter/main.tf]: Ok i was able to break out the various components.

File: terraform-kickstarter/prowler_build_spec.yml (hunk 1 above)
Comment (w0rmr1d3r) [reply terraform-kickstarter/prowler_build_spec.yml]: perfect!

File: terraform-kickstarter/main.tf (hunk 2 above)
Comment (w0rmr1d3r) [reply terraform-kickstarter/main.tf]: oh! nice! I saw the changes, much better now!

**Comment by toniblyx** [comment]
Awesome work @singergs, thank you!

**Pull request merged**